DEV/org.openl.rules/src/org/openl/rules/lang/xls/TableSyntaxNodeUtils.java

```diff
@@ -111,6 +111,7 @@ public static String str2name(String methodHeader, XlsNodeTypes tableType) {
                             XlsNodeTypes.XLS_CONDITIONS) || tableType.equals(XlsNodeTypes.XLS_ACTIONS) || tableType
                                 .equals(XlsNodeTypes.XLS_RETURNS) || tableType.equals(
                                     XlsNodeTypes.XLS_ENVIRONMENT) || tableType.equals(XlsNodeTypes.XLS_PROPERTIES)) {
+            resultName = StringUtils.normalizeSpace(resultName);

```

Comment:
1. This fix does not cover `Datatype` tables and not recognized tables
2. This fix does not covered by a test
3. Using StringUtils.normalizeSpace introduces memory consumption due needless arrays creation (see implementation)